satellite_determination/custom_dataclasses/overhead_window.py

```diff
@@ -8,3 +8,7 @@
 class OverheadWindow:
     satellite: Satellite
     overhead_time: TimeWindow
+
+    def __int__(self, satellite: Satellite, overhead_time: TimeWindow):
+        super().__init__(satellite, overhead_time)
+        assert overhead_time.timezone == utc
```

Comment:
This needs to be tested. Also probably don't ensure that the timezone is utc, but rather just that it has a timezone.

Reply:
Deleted, as it is unnecessary as UTC is always used when OverheadWindows are declared in program.

Reply:
I believe the point of this was to ensure nobody accidentally created one without a timezone, including users who are using this package. If you want to punt to another time, though, that's up to you.